purview/utils/general.py

```diff
@@ -153,18 +138,21 @@ def create_query(item, uid_name, uid, reverse=False):
 
     while story_flow[curr_node_label][rel_label]:
         if curr_node_label == item.__label__:
-            node = node_query(curr_node_label, uid_name, uid)
-        else:
-            node = node_query(curr_node_label)
+            query += 'MATCH ({0}:{1} {{{2}:"{3}"}})\n'.format(
```

Comment:
It'd be easier to read if you did:
```python
query = """\
    MATCH ({var}:{label} {{{uid_name}:"{uid}"}})
    CALL apoc.path.expandConfig({var}, {{sequence:\'{label}
""".format(var=curr_node_label.lower(), label=curr_node_label, uid_name=uid_name.rstrip('_'), uid=uid)
```

Reply:
You could search by the Neo4j internal ID instead of by the uid. That would simplify the amount of arguments you need to pass and it's like 1ms faster too (not actually important haha)!